widgets/social-media-buttons/data/networks.php

```diff
@@ -148,6 +148,12 @@
 		'icon_color' => '#FFFFFF',
 		'button_color' => '#B2C01C'
 	),
+	'discord'   => array(
+		'label'    => __( 'Discord', 'so-widgets-bundle' ),
+		'base_url' => 'https://discord.gg/',
```

Comment:
Is the domain not .com?

Reply:
Discord invite links are .gg

https://support.discord.com/hc/en-us/articles/204155938-How-do-I-invite-friends-to-my-server-